### ISSUE TITLE
Change icon to in '/user/share/applications/user-switch.desktop'

### ISSUE THREAD

**Issue by triantares** [Elive/elive-tools#22]
Current icon is used for other desktops.
Solve by changing line 85 to:
icon=system-switch-user

**Comment by triantares** [comment]
I cannot find the 'elive-skel.desktop' file or origin but that requires "icon=preferences-desktop-use" change.

**Comment by Thanatermesis** [comment]
> icon=system-switch-user

this icon is only available on the beautyline theme, no icon named like this on the default elive icon theme

(procedure should be the inverse, search an icon that exists in the default elive theme and see if exists too in the alternative ones like beautyline)

in fact the elive icon theme is very limited in variety, many icons are already used in other places, there's nothing wrong about this in fact it only removes some specific-personality for that specific app but if is not an important app, the important thing is to be intuitive about "what is doing"

> I cannot find the 'elive-skel.desktop' file here or the origin but that requires "icon=preferences-desktop-user" change.

```
~ ❯❯❯ dpkg -S elive-skel.desktop
elive-skel: /usr/share/applications/elive-skel.desktop
elive-center: /usr/share/elive-center/links/elive-skel.desktop
```

but, same issue, we don't have an icon named like that in the default elive icon theme

**Comment by triantares** [comment]
That is because the default Elive icon-theme misses a lot of icons, it hasn't been updated in years.
 If you look at other themes you'll see they are there in one form or another.

Yeah, I know that package but I was looking for the git repository.  That was the 'here' in my comment.

Anyway the 'tooltips' are inmportant too.

**Comment by Thanatermesis** [comment]
> That is because the default Elive icon-theme misses a lot of icons, it hasn't been updated in years.

Exactly, is difficult to find icons that looks the same designs, thats why is limited and I try to use existing icons as much as possible

**Comment by triantares** [comment]
Yeah, but that can be solved by adding an icon in the app itself.
And now it's coming to bite us in the elive-center on RetroWave where the icons are shown together.

At that moment the tooltip gets important and that isn't set correctly set on 'user-switch'. At least repair that.

**Comment by triantares** [comment]
Anyway setting those doesn't pose any real issues with "elive-default-icons" except maybe the elive-center in the dock.
<img src=https://i.imgur.com/f7OJ4rO.png >
